packages/accounts/src/light-account/types.ts

```diff
@@ -26,13 +26,17 @@ export type AccountVersionDef<
   >;
 };
 
-export type LightAccountVersion<
+export type LightAccountVersion =
+  | keyof IAccountVersionRegistry["LightAccount"]
+  | keyof IAccountVersionRegistry["MultiOwnerLightAccount"];
+
+export type GetLightAccountVersion<
```

Comment:
nit: why would we name a type with the prefix `Get...` ? shouldn't we just replace the previous `LightAccountVersion` type

Reply:
Because LightAccountVersion is already defined as actual type definition for type with input type param being all specified. This type helper infers or extracts the lightaccountversion that is tightly coupled with the input type param .